src/e2e/step-definitions/external-systems.ts

```diff
@@ -0,0 +1,91 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+import { Then, When } from "@cucumber/cucumber";
+import { Page } from "@playwright/test";
+import { CustomWorld } from "../support/worlds/world";
+
+const ONE_MINUTE_IN_MS = 60_000;
+
+// Reference to smartdocuments page
+let smartDocumentsPage: Page;
```

Comment:
```suggestion
let createSdDocumentPopup: Page;
```
Maybe another name is even better?

Reply:
I called it `smartDocumentsWizardPage` - the 'popup' is just how Playwright calls the event; tmo confusing to put that in the name.

Reply:
OK to close this @bas-info-nl  ?